omod/src/main/webapp/pages/reports/activeVisitsList.gsp

```diff
@@ -1,6 +1,5 @@
 <%
-    ui.decorateWith("appui", "standardEmrPage")
-    ui.includeCss("pihcore", "inpatient.css")
+    ui.decorateWith("appui", "standardEmrPage", [ includeBootstrap: false ])
```

Comment:
Can we include a comment here saying that we are excluding the version of bootstrap provided by the UI Framework core in order to use a more recent version that we are providing (and list both versions) just as a reminder when we need to edit/debug this the future to make it clear which version of bootstrap we are using)(=